src/ast/ddl.rs

```diff
@@ -143,6 +145,17 @@ pub enum AlterTableOperation {
         partition: Partition,
         with_name: Option<Ident>,
     },
+    /// `UPDATE <column> = <expression> [, ...] [IN PARTITION partition_id] WHERE <filter_expr>` 
+    /// Note: this is a ClickHouse-specific operation, please refer to
+    /// [ClickHouse](https://clickhouse.com/docs/en/sql-reference/statements/alter/update)
+    UpdateData {
```

Comment:
```suggestion
    Update {
```
Thinking we can call it update without the data suffix which doesn't seem to add significant context?

Reply:
ok. Renamed.